app/web/features/search/MapWrapper.tsx

```diff
@@ -0,0 +1,287 @@
+import maplibregl, { EventData, Map as MaplibreMap } from "maplibre-gl";
+import { addClusteredUsersToMap, layers } from "../search/users";
+import { reRenderUsersOnMap } from "features/search/users";
+import { useCallback, useEffect, useState } from "react";
+import ReplayIcon from "@material-ui/icons/Replay";
+import { Dispatch, SetStateAction } from "react";
+import { UserSearchRes } from "proto/search_pb";
+import TuneIcon from "@material-ui/icons/Tune";
+import { filterData } from "../search/users";
+import { InfiniteData } from "react-query";
+import makeStyles from "utils/makeStyles";
+import { usePrevious } from "utils/hooks";
+import { MutableRefObject } from "react";
+import Button from "components/Button";
+import { User } from "proto/api_pb";
+import Map from "components/Map";
+import { Point } from "geojson";
+import { SEARCH } from "i18n/namespaces";
+import { useTranslation } from "i18n";
+
+const useStyles = makeStyles((theme) => ({
+  searchHereGroup2: {
+    top: "20px",
+    // display: "flex",
```

Comment:
Do you want this commented out or did you mean to remove?

Reply:
remove, will update it :)

Reply:
done ✅